tools/capistrano/config/deploy.rb

```diff
@@ -14,4 +17,15 @@
 set :linked_files, [ ]
 set :linked_directories, [ ]
 
+# Slack notifications
+set :domain, '{{hostname}}'
+set :slack_channel, '#{{name}}'
+set :github_url, 'https://github.com/{{git_url}}'.gsub!('git@github.com:', '')
+set :jira_url, 'https://jira.example.com/'
+set :jira_project, '\\(PROJECT1|PROJECT2|PROJECT3\\)' # regular expression!
+set :slack_hook_url, 'https://hooks.slack.com/services/.../.../...'
```

Comment:
as devs are inevitably going to forget some of these settings, I think it might be better to exclude the examples (or just comment out) and have errors if not set